JsonLd/Event/ContextBuilderEvent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Dunglas\ApiBundle\JsonLd\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+class ContextBuilderEvent extends Event
+{
+    private $context;
+
+    public function __construct(array $context) {
+
+        $this->context = $context;
+    }
+
+    /**
+     * @return array
+     */
+    public function getContext()
+    {
+        return $this->context;
+    }
+
+    /**
+     * @param array $context
+     *
+     * @return array
+     */
+    public function setContext(array $context)
+    {
+        $this->context = $context;
+    }
+
+    /**
+     * @param $key
+     * @param $value
+     *
+     * @return ContextBuilderEvent
+     */
+    public function addToContext($key, $value)
```

Comment:
I agree.
I also suggest to name it ResourceContext or something similar, as it is confusing to use context (array) both for the serializer and Json-Ld